config/checkstyle_checks.xml

```diff
@@ -394,6 +394,10 @@
     </module>
     <module name="InnerAssignment"/>
     <module name="MagicNumber"/>
+    <module name="MatchXpath">
+      <property name="query" value="//CLASS_DEF[@text!='Checker' and @text!='Main']
+      //LITERAL_CATCH//METHOD_CALL[.//IDENT[@text = 'printStackTrace']]/.."/>
```

Comment:
> Illegal code structure detected.

Can we add a message and explain what is illegal and our advice to resolve the issue. It is fine if it is just "Avoid using 'printStackTrace'.`

Reply:
done

Reply:
@rnveach , very good catch.
I moving discussion to https://github.com/checkstyle/checkstyle/pull/8902#pullrequestreview-542906795 to update more, as it is very critical item for Check usability.